meveo-admin/ejbs/src/main/java/org/meveo/service/base/BusinessService.java

```diff
@@ -240,8 +250,12 @@ public void removeFilesFromModule(P entity, MeveoModule module) throws BusinessE
     			throw new BusinessException("Folder unsuccessful deleted : " + directoryToRemove.getPath() + ". " + e.getMessage(), e);
     		}
     	}
+//    	gitClient.commitFiles(meveoRepository, Collections.singletonList(directoryToRemove), "Remove directory " + directoryToRemove.getPath());
+    	
+    	List<String> pattern = new ArrayList<String>();
+    	pattern.add(GitHelper.computeRelativePath(gitDirectory, directoryToRemove));
+    	gitClient.commit(meveoRepository, pattern, "Remove directory " + directoryToRemove.getPath());
```

Comment:
You should not use the meveoRepository, but rather the repository of the concerned module